lib/new_relixir/stats.ex

```diff
@@ -14,17 +14,18 @@ defmodule NewRelixir.Stats do
     {[webtransaction_total(ms), db_total(ms) | errors_total(errs) ++ ms], errs, time}
   end
 
+  def transform_error_counter({{scope, {type, message}}, count}), do: transform_error_counter({{scope, type, message}, count})
```

Comment:
why two heads here? would it be more clear to transform these inputs when this function is called?